src/content/capture/canvas-video/js/main.js

```diff
@@ -0,0 +1,18 @@
+/*
+*  Copyright (c) 2016 The WebRTC project authors. All Rights Reserved.
+*
+*  Use of this source code is governed by a BSD-style license
+*  that can be found in the LICENSE file in the root of the source
+*  tree.
+*/
+
+'use strict';
+
+// Call main() in demo.js
+main();
+
+var canvas = document.querySelector('canvas');
+var video = document.querySelector('video');
+
+var stream = canvas.captureStream();
+video.src = URL.createObjectURL(stream);
```

Comment:
D'oh! Force of habit. Fixed.

On 8 May 2016 8:14 a.m., "Philipp Hancke" <notifications@github.com> wrote:

> In src/content/capture/canvas-video/js/main.js
> <https://github.com/webrtc/samples/pull/784#discussion_r62427985>:
>
> > +*
> > +*  Use of this source code is governed by a BSD-style license
> > +*  that can be found in the LICENSE file in the root of the source
> > +*  tree.
> > +*/
> > +
> > +'use strict';
> > +
> > +// Call main() in demo.js
> > +main();
> > +
> > +var canvas = document.querySelector('canvas');
> > +var video = document.querySelector('video');
> > +
> > +var stream = canvas.captureStream();
> > +video.src = URL.createObjectURL(stream);
>
> can't you just use srcObject =?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/webrtc/samples/pull/784/files/a2024b1d3a40942f19311b34d7d1626ecfeaaf7e#r62427985>
>